kafka/src/main/java/io/micronaut/configuration/kafka/processor/KafkaConsumerProcessor.java

```diff
@@ -174,7 +179,8 @@ class KafkaConsumerProcessor
             ProducerRegistry producerRegistry,
             KafkaListenerExceptionHandler exceptionHandler,
             @Named(TaskExecutors.SCHEDULED) ExecutorService schedulerService,
-            TransactionalProducerRegistry transactionalProducerRegistry) {
+            TransactionalProducerRegistry transactionalProducerRegistry,
+            ApplicationEventPublisher eventPublisher) {
```

Comment:
We should inject two publishers:

- `ApplicationEventPublisher<KafkaConsumerStartedPollingEvent> kafkaConsumerStartedPollingEventPublisher`
- `ApplicationEventPublisher<KafkaConsumerSubscribedEvent> kafkaConsumerStartedPollingEventPublisher`

Reply:
@guillermocalvo please do this change